site/sfguides/src/getting_started_with_native_apps/getting_started_with_native_apps.md

```diff
@@ -314,233 +405,111 @@ grant usage on streamlit app_instance_schema.streamlit to application role app_i
-## Upload Consumer Supply Chain Data
-Duration: 2
-
-In this scenario, consumers will provider their own supply chain data (orders and site recovery data) from their own Snowflake account. The app will use the consumer's data to render graphs representing different aspects of the supply chain.
+We've now added some basic functionality to the native app. From here, we will create the first version of our application package. You can have multiple versions available. 
 
-Run the SQL below in a worksheet. We'll use the `NATIVE_APP_QUICKSTART_DB` to store the consumer supply chain data.
+Yo create our application package version we are going to execute the following command:
 
 ```
-USE WAREHOUSE NATIVE_APP_QUICKSTART_WH;
-
--- this database is used to store our data
-USE DATABASE NATIVE_APP_QUICKSTART_DB;
-
-USE SCHEMA NATIVE_APP_QUICKSTART_SCHEMA;
-
-CREATE OR REPLACE TABLE MFG_ORDERS (
-  order_id NUMBER(38,0), 
-  material_name VARCHAR(60),
-  supplier_name VARCHAR(60),
-  quantity NUMBER(38,0),
-  cost FLOAT,
-  process_supply_day NUMBER(38,0)
-);
-
--- Load app/data/orders_data.csv using Snowsight
-
-CREATE OR REPLACE TABLE MFG_SITE_RECOVERY (
-  event_id NUMBER(38,0), 
-  recovery_weeks NUMBER(38,0),
-  lat FLOAT,
-  lon FLOAT
-);
-
--- Load app/data/site_recovery_data.csv using Snowsight
+snow app version create V1 --interactive --database NATIVE_APP_QUICKSTART_PACKAGE
 ```
 
-### Upload data into these tables  
+This code is going to create the Version number 1 of out application package, in the NATIVE_APP_QUICKSTART_PACKAGE 
+database, the --interactive directive is used to show more information in other prompts different than terminal devices.
+As we explained before, you can also set the version information inside the manifest.yml, therefore when you run the `snow app run` command, it is going to automatically create the versioned application. It would look something like this:
 
-Next, you'll load data into these newly defined tables using the Snowflake UI.
-
-1. Login to your Snowflake account and navigate to `Data -> Databases`
-2. Select `NATIVE_APP_QUICKSTART_DB` -> `NATIVE_APP_QUICKSTART_SCHEMA` -> `Tables` 
-3. From there, select each table (`MFG_ORDERS`, `MFG_SITE_RECOVERY` ) and upload the corresponding `.csv` file from the cloned repository:
-   1. Select `Load Data` in the top right
-   2. Select `NATIVE_APP_QUICKSTART_WH` for the warehouse
-   3. Click `Browse` and select the corresponding `.csv` file, Click `Next`
-   4. Select `Delimited Files (CSV or TSV)` for the File Format
-   5. For `Field optionally enclosed by` select `Double quotes`
-   6. Click `Next`, repeat for each table. 
+```
+version:
+  name: V1
+  label: Version One
+  comment: The first version of the application
+```
+In this case, because the first version was already created, the system is going to add a patch version by default to make the differentiation from both packages, but this could be explicitly set in the `snow app version create` options.
```

Comment:
```suggestion
Because the first version was already created, the system is going to add a new patch in the existing version. You can explicitly number the patch by adding `--patch <number>` to the `snow app version create` arguments.
```